features/searches.feature

```diff
@@ -451,3 +451,11 @@ Feature: Search
     And I search for "qeury"
     Then I should see "Showing results for query"
     And I should see "Search instead for qeury"
+
+  @javascript
+  Scenario: Search with a blank query on an affiliate page (redesign)
+    Given the following Affiliates exist:
+      | display_name     | name             | contact_email         | first_name | last_name |
+      | bar site         | bar.gov          | aff@bar.gov           | John       | Bar       |
+    When I am on bar.gov's redesigned search page with "" query
+    Then I should see "Please enter a search term in the box above."
```

Comment:
Thanks for reviewing. I agree that a new feature file should be used. The test will require some tweaking since the search box does not work in the dummy UI; right now, the search query has to be passed in through the URL.